package.json

```diff
@@ -51,7 +51,17 @@
     "package": "rm -rf dist && copyfiles -u 1 'src/**/*' dist",
     "test": "react-app-rewired test",
     "eject": "react-scripts eject",
-    "prepublishOnly": "npm run package"
+    "prepublishOnly": "npm run package",
+    "prepare": "husky install",
+    "pre-commit": "lint-staged"
```

Comment:
Do you actually need this line for pre-commit hook to work? According to husky [docs](https://github.com/typicode/husky/tree/main/docs#create-a-hook) that doesn't seem as a prerequisite.

Reply:
No, it really works by default